ok-marketplace-be/ok-marketplace-stubs/build.gradle.kts

```
@@ -0,0 +1,36 @@
+plugins {
+    kotlin("multiplatform")
```

Здесь что-то накосячено.
У нас сейчас кругом подключается кастомный плагин
```suggestion
    id("build-kmp")
```